src/trait_.rs

```diff
@@ -201,6 +210,7 @@ pub trait TraitImpl {
 	/// Build method body for this [`Trait`].
 	fn build_body(
 		&self,
+		_any_bound: bool,
```

Comment:
Could have also passed `&[Generics]` directly, but I guess we can change that when we actually need it.

Reply:
yeah, I think this should be fine for now.